packages/fluent-ui/src/TextWidget/TextWidget.tsx

```diff
@@ -84,7 +84,7 @@ const TextWidget = ({
       // TODO: once fluent-ui supports the name prop, we can add it back in here.
       // name={name}
       type={inputType as string}
-      value={value ? value : ""}
+      value={typeof value === "undefined" || value === null ?  "" : value}
```

Comment:
Actually, let's just do the same thing as in https://github.com/rjsf-team/react-jsonschema-form/blob/110adbe1b1d46edbae512c22dfd1f8d5183ceba1/packages/material-ui/src/TextWidget/TextWidget.tsx#L59:

```suggestion
      value={value || value === 0 ? value : ""}
```